src/property-names.js

```diff
@@ -15,12 +15,14 @@
 (function(scope, testing) {
 
   var aliased = {};
+  var canonical = {};
```

Comment:
I find the name "canonical" not very obvious, I have to read the logic to understand what it means. Perhaps renaming them as s/aliased/prefixed/ and s/canonical/unprefixed/ would be clearer.